run_local_with_dependencies.sh

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env bash
 
-sm --start API_DOCUMENTATION SERVICE_LOCATOR API_DEFINITION API_EXAMPLE_MICROSERVICE -f
```

Comment:
Why have you removed this?

Reply:
Mistake. Changed it back again with Dave's help